client/app/components/form/SubmitButton.tsx

```diff
@@ -5,28 +5,23 @@ import { useFormStatus } from "react-dom";
 
 interface SubmitButtonProps {
   label: string;
-  classNames?: string;
   disabled?: boolean;
 }
 
 const SubmitButton: React.FunctionComponent<SubmitButtonProps> = ({
   label,
-  classNames,
   disabled,
 }) => {
   const { pending } = useFormStatus();
   return (
-    <div className={classNames}>
-      <Button
-        variant="contained"
-        type="submit"
-        aria-disabled={disabled || pending}
-        className="h-full"
-        sx={{ marginBottom: 10 }}
```

Comment:
@BCerki I think this `sx={{ marginBottom: 10 }}` is another great example of why basic reusable components like buttons shouldn't be wrapped in divs and have margins applied to them. This `marginBottom` was the cause of all those big buttons here and the `select operator` page and caused other weird issues.

Of course this isn't a hard rule and a best practice. There are exceptions such as something like a `Header` component where you may want the same margins throughout the page for consistency.